commands/config.js

```diff
@@ -136,15 +136,24 @@ function printHelp () {
   helpHeader(
     'config',
     chalk`ncm {${COLORS.yellow} config} {${COLORS.teal} <action>}`,
-    'ncm config <action>'
+    'ncm config <action>',
+    `
+Allows easy adjusting of ncm-cli configuration and local storage.
```

Comment:
```suggestion
Allows adjusting of ncm-cli configuration and local storage.
```

Not sure we need this subjectivity here.